render-doc

```diff
@@ -4,14 +4,32 @@ go build -o gotemplate
 
 CONTENT_FOLDER="docs/content"
 DOC_FOLDER="$CONTENT_FOLDER/docs"
-export COLUMNS=1024
+export COLUMNS=108
 
 rm -rf $DOC_FOLDER
 mkdir -p $DOC_FOLDER
 
+git submodule update --init
+
+echo '
+.flink {
+    display: inline-block;
+    min-width: 2.5in;
+}
+' > docs/themes/book/assets/_custom.scss
```

Comment:
You don't have to put that in the book theme directly. You create an `assets` folder beside content. We don't need to generate this dynamically. Just commit the scss file

Reply:
https://github.com/alex-shpak/hugo-book#extra-customisation

Reply:
OK, thanks.